doc/source/conf.py

```diff
@@ -77,9 +80,27 @@ def remove_doctree(app, exception):
     shutil.rmtree(app.doctreedir)
 
 
+def copy_examples(app):
+    """Copy directory examples (root directory) files into the doc/source/examples directory.
+    """
+    DESTINATION_DIRECTORY = pathlib.Path(app.srcdir, "examples").resolve()
+    print(f"Copying examples from {EXAMPLES_DIRECTORY} to {DESTINATION_DIRECTORY} directory")
```

Comment:
It would be great to use here the logging from Sphinx. This way, this message would be collected in the log files and during the output.

Reply:
https://www.sphinx-doc.org/en/master/extdev/logging.html